### DISCUSSION TITLE
Document and script the release process

### DISCUSSION THREAD

**Discussion by leonchabbey** [react-native-tvos/react-native-tvos#363] [Feature Requests]
### Description

I don't see anything in the README about it and it looks like you are using circleci for releases like the official react-native repo.

I tried following the instructions found here: [Publish your own version of React Native ](https://reactnative.dev/contributing/how-to-build-from-source#publish-your-own-version-of-react-native), but the docker freezes during the process for an unknown reason.

Does it work for you or do you have any other idea how I could easily do it?

### Version

0.68-2

### Output of `npx react-native info`

OS: macOS 12.3.1
CPU: (8) arm64 Apple M1
Memory: 119.30 MB / 16.00 GB
Shell: 5.8 - /bin/zsh

**Comment by douglowder** [comment]
I do not use Docker or CircleCI for this.

If you need to make a release tarball locally, you should do the following from the top level `react-native-tvos` directory:

```sh
./gradlew :ReactAndroid:installArchives
npm pack
```
This generates the prebuilt Maven archive for Android, and then packages up all the files specified in `package.json`.

**Comment by leonchabbey** [comment]
Great it worked ! Thanks for the info 🙂 

Maybe a release section could be added to the documentation?

**Comment by douglowder** [comment]
Sure! Ideally this should be scripted so I'll turn this into a feature request for that, plus add the docs.

**Comment by douglowder** [comment]
I have added some release preparation scripts for TV in https://github.com/react-native-tvos/react-native-tvos/commit/e2d3f7a24ac070a76bd63f4e109cd7793ebafa00 -- still a work in progress but very much improved.
